File: srm174/cross_word.py
class CrossWord:
	def countWords(self, board, size):
		words = []
		for i in range(len(board)):
			tmp = 0
			for j in range(len(board[i])):
				if board[i][j] == '.': tmp += 1
				else:
					words.append(tmp)
					tmp = 0
			words.append(tmp)
		return words.count(size)

File: srm174/CrossWord.java
import java.util.*;

public class CrossWord {

	public int countWords(String[] board, int size) {
		int count = 0;
		char dot = '.';
		for (int i = 0; i < board.length; i++) {
			int tmp = 0;
			for (int j = 0; j < board[i].length(); j++) {
				if (board[i].charAt(j) == dot) {
					tmp += 1;
				} else {
					if (tmp == size) {count++;}
					tmp = 0;
				}			
				
			}
			
			if (tmp == size) {
				count++;
			}
		}
			
		return count;
	}

 }